Makefile

```diff
@@ -55,10 +55,12 @@ pd-recover:
 	CGO_ENABLED=0 go build -o bin/pd-recover tools/pd-recover/main.go
 
 test: retool-setup
-	# testing..
+	# testing...
+	make deadlock-enable
 	@$(FAILPOINT_ENABLE)
-	CGO_ENABLED=1 GO111MODULE=on go test -race -cover $(TEST_PKGS) || { $(FAILPOINT_DISABLE); exit 1; }
+	CGO_ENABLED=1 GO111MODULE=on go test -race -cover $(TEST_PKGS) || { $(FAILPOINT_DISABLE); make deadlock-disable; exit 1; }
 	@$(FAILPOINT_DISABLE)
+	make deadlock-disable
```

Comment:
I wonder is it a good practice to use make in Makefile?

Reply:
OK, I will find another way to do it.